core/ecschema-editing/src/Validation/SchemaCompareVisitor.ts

```diff
@@ -20,21 +21,28 @@ import { ISchemaComparer, SchemaComparerOptions } from "./SchemaComparer";
 export class SchemaCompareVisitor implements ISchemaPartVisitor {
   private _schemaB: Schema;
   private _schemaComparer: ISchemaComparer;
-  private _lookupFullName: boolean;
 
   /**
    * Initializes a new SchemaCompareVisitor instance.
    * @param schemaComparer The [[SchemaComparer]] to use to compare each item of the schema.
    * @param schemaToCompare The second or 'B' schema to compare against the schema being traversed (Schema A).
    */
-  constructor(schemaComparer: ISchemaComparer, schemaToCompare: Schema, options?: SchemaComparerOptions) {
+  constructor(schemaComparer: ISchemaComparer, schemaToCompare: Schema) {
     this._schemaComparer = schemaComparer;
     this._schemaB = schemaToCompare;
-    this._lookupFullName = options?.compareItemFullName ?? true;
   }
 
+  /**
+   * Called to look up higher level schema items.
+   * @param item
+   * @returns
+   */
   private async lookupItem<T extends SchemaItem = SchemaItem>(item: SchemaItem) {
-    return this._schemaB.lookupItem<T>(this._lookupFullName ? item.key : item.name);
+    let key: SchemaItemKey | string = item.key;
+    if (item.schema.name !== this._schemaB.name){
```

Comment:
I'm walking through the uses of this method and it looks like I was wrong ... it looks like it is only used to look up items directly declared in schema a.  Do you agree?  If so then we can always just use the name.

Reply:
Function has been removed and now we use name get items.